### PR TITLE
[v6r14] Fix sending mail to multiple recipients

### LINKED ISSUE #3007: Sending mail to multiple recipients does not work
Sending mail to multiple recipients (e.g. from Bdii2CSAgent) does not work. Only the first recipient will receive the mail.

Looks like a bug in the utility module DIRAC/Core/Utilities/Mail.py to me. It expects the recipients either as a list or as a string. However, the other modules simply take the value of "MailTo" from the CS - which is a comma-separated string in case of multiple values. DIRAC/Core/Utilities/Mail.py contains a check for that:

```
if not type( self._mailAddress ) == type( [] ):
    addresses = [self._mailAddress]
```

However, the solution should look like this to properly handle comma-separated recipients:

```
if not type( self._mailAddress ) == type( [] ):
    addresses = self._mailAddress.split(", ")
```
Do you agree here?

### PR DESCRIPTION
**Pull request by ahaupt**
Should fix #3007

### CODE REVIEW

File: Core/Utilities/Mail.py [hunk 1]
```diff
@@ -33,8 +33,8 @@ def _send( self ):
 
     mail = MIMEText( self._message , "plain" )
     addresses = self._mailAddress
-    if not type( self._mailAddress ) == type( [] ):
-      addresses = [self._mailAddress]
+    if isinstance( self._mailAddress, basestring):
+        addresses = self._mailAddress.split(", ")
```
Comment (chaen) [inline Core/Utilities/Mail.py]: We use two spaces as the default indentation

Comment (ahaupt) [inline Core/Utilities/Mail.py]: you're the boss :-)

Comment (chaen) [inline Core/Utilities/Mail.py]: Haha, I'm going to make a screenshot of that one, it can show useful one day :D

**Comment by fstagni** [comment]
Review OK

**Pull request merged**